RHDEVDOCS-4137 - Added 1.5.1 and 1.5.2 release notes (#47155)
openshift/openshift-docs: modules/gitops-release-notes-1-5-1.adoc

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

> 

Reviewer: ```suggestion
:_content-type: REFERENCE
```
Every file must have a content type attribute.

[id="gitops-release-notes-1-5-1_{context}"]
= Release notes for {gitops-title} 1.5.1

{gitops-title} 1.5.1 is now available on {product-title} 4.8, 4.9, and 4.10.

[id="fixed-issues-1-5-1_{context}"]
== Fixed issues

The following issues have been resolved in the current release:

> * Before this update, an unauthenticated user can craft a JWT token while Argo CD's anonymous access is enabled and get full access to the Argo CD instance. This issue is fixed now. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-29165]

Reviewer: ```suggestion
* Before this update, an unauthenticated user was able to craft a JWT token while Argo CD's anonymous access is enabled and get full access to the Argo CD instance. This issue is fixed now. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-29165]
```

Reviewer: Please review this suggestion.

> * Before this update, an unauthenticated user can display error messages on the login screen when SSO in enabled. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081689[CVE-2022-24905] 

Reviewer: ```suggestion
* Before this update, an unauthenticated user was able to display error messages on the login screen while SSO was enabled. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081689[CVE-2022-24905] 
```

> * Before this update, all unpatched versions Argo CD starting v7.0 were vulnerable to a symlink following bug. This allowed an unauthorized user with repository write
> access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]

Reviewer: ```suggestion
* Before this update, all unpatched versions Argo CD starting v7.0 were vulnerable to a symlink-following bug. This allowed an unauthorized user with repository write access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]
```

Reviewer: Note: `symlink-following` and `...fixed.  link:...`

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

:_content-type: REFERENCE

[id="gitops-release-notes-1-5-1_{context}"]
= Release notes for {gitops-title} 1.5.1

{gitops-title} 1.5.1 is now available on {product-title} 4.8, 4.9, and 4.10.

[id="fixed-issues-1-5-1_{context}"]
== Fixed issues

The following issues have been resolved in the current release:

> * Before this update, an unauthenticated user was able to craft a JWT token while Argo CD's anonymous access is enabled and get full access to the Argo CD instance. This issue is fixed now. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-29165]

Reviewer: Please see https://github.com/openshift/openshift-docs/pull/47155#discussion_r908350064

* Before this update, an unauthenticated user was able to display error messages on the login screen while SSO was enabled. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081689[CVE-2022-24905] 

> * Before this update, all unpatched versions of Argo CD starting v7.0 were vulnerable to a symlink-following bug. This allowed an unauthorized user with repository write access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]

Reviewer: ```suggestion
* Before this update, all unpatched versions of Argo CD v7.0 and later were vulnerable to a symlink-following bug. This bug made it possible for an unauthorized user with repository write access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]
```
